src/components/02-organisms/Attachments/index.js

```diff
@@ -178,6 +179,7 @@ const Attachments = React.memo(props => {
   const [fileError, setFileError] = useState(false);
   const [isNameModalOpen, setIsNameModalOpen] = useState(false);
   const [nameMissing, setNameMissing] = useState(false);
+  const [showFileDownloader, setFileDownloader] = useState(false);
```

Comment:
setFileDownloader should be setShowFileDownloader. Actually would it be better to name the variable as [isFileDownloaderVisible, setIsFileDownloaderVisible]?